build.gradle

```diff
@@ -42,93 +41,80 @@ repositories {
+    implementation group: 'bio.terra.cloud-resource-lib', name: 'google-bigquery', version: '0.13.0'
+    implementation group: 'bio.terra.cloud-resource-lib', name: 'google-notebooks', version: '0.10.0'
+    implementation group: 'bio.terra.cloud-resource-lib', name: 'google-storage', version: '0.14.0'
+
+    // hk2 is required to use WSM client, but not correctly exposed by the client
+    implementation group: 'org.glassfish.jersey.inject', name: 'jersey-hk2', version: '2.36' // TODO
+
+    // command parsing
+    implementation group: 'info.picocli', name: 'picocli', version: '4.6.3'
+
+    // logging
+    implementation group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.11'
+    implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.36'
+
+    // serialization
+    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.13.3'
+    implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.13.3'
+
+    // GCP
+    implementation group: 'com.google.auth', name: 'google-auth-library-credentials', version: '1.10.0'
+    implementation group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version: '1.10.0'
+    implementation group: 'com.google.oauth-client', name: 'google-oauth-client-java6', version: '1.34.1'
+    implementation group: 'com.google.oauth-client', name: 'google-oauth-client-jetty', version: '1.34.1'
+    implementation group: 'com.google.cloud', name: 'google-cloud-bigquery', version: '2.14.6'
+    implementation group: 'com.google.cloud', name: 'google-cloud-storage', version: '2.11.3'
+
+    // Docker
+    implementation group: 'com.github.docker-java', name: 'docker-java-core', version: '3.2.13'
+    implementation group: 'com.github.docker-java', name: 'docker-java-transport-httpclient5', version: '3.2.13'
+
+    // static analysis
+    compileOnly group: 'com.github.spotbugs', name: 'spotbugs-annotations', version: '4.7.1'
+    testCompileOnly group: 'com.github.spotbugs', name: 'spotbugs-annotations', version: '4.7.1'
+
+    // Deps whose versions are controlled by Spring
+    implementation group: 'org.apache.commons', name: 'commons-text', version: '1.9'
+    implementation group: 'org.apache.commons', name: 'commons-lang3'
+
+    // Swagger deps
+    implementation group: 'io.swagger.core.v3', name: 'swagger-annotations', version: '2.2.2'
+
+    // Test deps
+    testImplementation platform('org.junit:junit-bom:5.9.0')
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter', version: '5.9.0'
+    testImplementation group: 'org.hamcrest', name: 'hamcrest', version: '2.2'
+
+    // PubSub is used in tests to communicate with Janitor, but is not required for the CLI itself.
+    testImplementation group: 'com.google.cloud', name: 'google-cloud-pubsub', version: '1.120.11'
+
+    annotationProcessor group: 'info.picocli', name: 'picocli-codegen', version: '4.6.3'
+}
+
+dependencyLocking {
```

Comment:
thanks for adding this and reordering all the dependency... but i prefer having one PR to do one thing especially since github diff viewer is crazy...

Reply:
Sure, I plan to get All tests working and then Split the changes - this will make it much easier to avoid breaking any of the tests